include/deal.II/lac/sparse_direct.h

```diff
@@ -291,9 +302,11 @@ class SparseDirectUMFPACK : public Subscriptor
 
   /**
    * The arrays in which we store the data for the solver.
+   * SuiteSparse_long has to be used here for Windows 64 build,
+   * if we used only long int, compilation would fail.
    */
-  std::vector<long int> Ap;
-  std::vector<long int> Ai;
+  std::vector<SuiteSparse_long> Ap;
+  std::vector<SuiteSparse_long> Ai;
   std::vector<double> Ax;
```

Comment:
Can you please open an issue (independent of this pull request) with a short statement that the type <code>SuiteSparse_long</code> might be different from <code>long int</code>.